### ISSUE TITLE
Rgb-depth pairs for fine-tuning

### ISSUE THREAD

**Issue by abhishekmonogram** [LiheYoung/Depth-Anything#56]
Hi @LiheYoung, I wanted to finetune the metric depth estimation code on my own dataset. I wanted to ask you about the number of pairs of rgb-depth required to get good metrics on metric depth estimation?  The NYUV2 dataset have ~36000 pairs of rgb-depth images in the training dataset. Do you think a few 100 or 1000  pairs maybe enough for accurate metric depth estimation?

**Comment by cosmosmosco** [comment]
Hello, may I ask which weight model you use for fine-tuning? I used `depth_anything_vitl14.pth` according to the [ReadMe](https://github.com/LiheYoung/Depth-Anything/tree/main/metric_depth), but there was a mismatch error in state_dict. Then I tried `depth _ anything _ metric _ depth _ outdoor.pt`, and there was no such error. I wonder if you did the same?

**Comment by oconnor127** [comment]
@abhishekmonogram I also want to fine-tune the model on a custom dataset. How did you do that?
